lib/iamport/version.rb

```diff
@@ -1,3 +1,3 @@
 module Iamport
-  VERSION = "0.2.4"
+  VERSION = "0.2.4".freeze
```

Comment:
It seems we should change the major version because it would not compatible with lower version method names.